libs/tup-components/src/allocations/AllocationTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { LoadingSpinner, InlineMessage } from '@tacc/core-components';
+import { useProjects } from '@tacc/tup-hooks';
+import { formatDate } from '../utils/timeFormat';
+
+export const AllocationsTable: React.FC = () => {
+  const { data, isLoading, error } = useProjects();
+
+  if (isLoading) {
+    return <LoadingSpinner />;
+  }
+
+  if (error) {
+    return (
+      <InlineMessage type="warning">
+        Unable to retrieve allocation information.
+      </InlineMessage>
+    );
+  }
+
+  return (
```

Comment:
Instead of a mega-table for all projects, this should probably be implemented as a table for a single allocation, using a [ProjectsAllocations](https://github.com/TACC/tup-ui/blob/9f5e633c609604758f32e57dcf4a35303ad27aed/libs/tup-hooks/src/index.ts#L83) object as a prop. We can go in later and write a layout component that takes the list of all projects and renders a separate table for each of them.